docs-web/configuration/security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+We consider the security of the Ingress Controller paramount to the success of our users and use the following to 
```

Comment:
"user" has been capitalized in other places in the doc, probably should be here as well.

Reply:
What do you think of:
"The security of the Ingress Controller is paramount to the success of our Users, and we use the following tools and processes to develop software as securely as possible."

Reply:
updated